templates/job-runner/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM liferay/base:latest
+
+ARG LABEL_BUILD_DATE
+ARG LABEL_NAME
+ARG LABEL_VCS_REF
+ARG LABEL_VCS_URL
+ARG LABEL_VERSION
+
+RUN apt-get install cron && \
+	apt-get clean
+
+COPY scripts/* /usr/local/bin/
+
+ENTRYPOINT ["/usr/local/bin/liferay_job_runner_entrypoint.sh"]
```

Comment:
```suggestion
ENV TINI_VERSION v0.19.0
ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
RUN chmod +x /tini

ENTRYPOINT ["/tini", "-v", "--", "/usr/local/bin/liferay_job_runner_entrypoint.sh"]
```

Reply:
hey @zsoltbalogh  here is a way that we can get the job-runner container to listen for SIGTERM events.  For reference: https://hynek.me/articles/docker-signals/